community/modules/scripts/htcondor-install/files/install-htcondor.yaml

```diff
@@ -19,13 +19,32 @@
     enable_docker: true
   become: true
   tasks:
-  - name: Setup HTCondor yum repository
-    ansible.builtin.yum:
+  - name: Enable EPEL repository
+    ansible.builtin.package:
       name:
       - epel-release
-      - https://research.cs.wisc.edu/htcondor/repo/10.x/htcondor-release-current.el7.noarch.rpm
+  - name: Enable HTCondor Feature Release repository
```

Comment:
nit: This looks potentially harder to maintain (by me 😄). Is there a reference on the internet that discusses this setup? or at least was used to derive this setup? Consider adding a link either as a comment or in the PR description.

Reply:
For better or for worse, the HTCondor team has switched to a [one script to rule them all](https://get.htcondor.com) approach to their primary instructions. I will link to the best available instructions as a comment in this YAML.

https://htcondor.readthedocs.io/en/latest/getting-htcondor/from-our-repositories.html